Makefile

```diff
@@ -39,15 +39,16 @@ VERSION ?= v0.0.0
 BUILD_TIMESTAMP ?= $(shell date +%s)
 REVISION ?= $(BRANCH)-$(COMMIT_SHORT_SHA)
 
-LDFLAGS := -X go.opentelemetry.io/ebpf-profiler/vc.version=$(VERSION) \
+LDFLAGS ?= -X go.opentelemetry.io/ebpf-profiler/vc.version=$(VERSION) \
```

Comment:
This may introduce indeterminism if `LDFLAGS` is set in the environment. Why is this needed here? One can still override LDFLAGS set with `:=` if one passes the value through a command-line argument, e.g.: make LDFLAGS=foo